module/VuFind/config/module.config.php

```diff
@@ -78,6 +78,16 @@
                     ]
                 ]
             ],
+            'legacy-holds' => [
+                'type' => 'Laminas\Router\Http\Literal',
+                'options' => [
+                    'route'    => '/MyResearch/Holds',
+                    'defaults' => [
+                        'controller' => 'Holds',
```

Comment:
How would you feel about continuing to point this at the MyResearchController's holdsAction, but instead changing that action to force a redirect to the new holds-list route? That way, people will get sent to the new URL instead of having two different URLs that do the same thing.

Reply:
(My thought is that eventually, after we complete refactoring, the MyResearchController will be nothing but a collection of redirects... and then we can deprecate and remove the whole thing).

Reply:
I'm ok with that if it's easier to deprecate and drop in future. I just tried to remove one hop from the execution, but your point is valid.